localisation/00_zuwc_united_localisation.csv

```diff
@@ -1209,7 +1209,7 @@ EVTDESC_WCTHW_2010;The people who have become infected with the plague cannot be
 EVTOPTA_WCTHW_2010;I will save them from their suffering!;;Ich werde sie von ihren Qualen erl�sen!;;;;;;;;;;;x
 EVTOPTB_WCTHW_2010;I'm not a butcher!;;Ich bin kein Schl�chter!;;;;;;;;;;;x
 EVTTITLE_WCTHW_2011;The Culling of [From.GetName];L��puration de [From.GetName];Die S�uberung [From.GetName]s;;;;;;;;;;;x
-EVTDESC_WCTHW_2011_world;Terrible news have has reached your court. The Scourge drives people to carry out atrocious acts... [Root.GetTitledFirstName] and [Root.GetHerHis] army have killed all of the infected residents of [From.GetName]. Among the infected people were men, women and children, but [Root.GetFirstName] showed no mercy. After the butchering, [Root.GetSheHe] set the city on fire and left it in ruins.;;Schreckliche Nachrichten haben Euren Hof erreicht. Die Gei�el bringt die Menschen dazu, schreckliche Dinge zu tun.\n[Root.GetTitledFirstName] und [Root.GetHerHis]e Armee haben die infizierten Einwohner [From.GetName] abgeschlachtet. Unter ihnen waren M�nner, Frauen und Kinder, aber [Root.GetFirstName] zeigte kein Erbarmen. Nach [Root.GetHerHis]er schrecklichen Tat setzte [Root.GetSheHe] die Stadt in Brand und legte sie in Schutt und Asche.;;;;;;;;;;;x
+EVTDESC_WCTHW_2011_world;Terrible news has reached your court. The Scourge drives people to carry out atrocious acts... [Root.GetTitledFirstName] and [Root.GetHerHis] army have killed all of the infected residents of [From.GetName]. Among the infected people were men, women and children, but [Root.GetFirstName] showed no mercy. After the butchering, [Root.GetSheHe] set the city on fire and left it in ruins.;;Schreckliche Nachrichten haben Euren Hof erreicht. Die Gei�el bringt die Menschen dazu, schreckliche Dinge zu tun.\n[Root.GetTitledFirstName] und [Root.GetHerHis]e Armee haben die infizierten Einwohner [From.GetName] abgeschlachtet. Unter ihnen waren M�nner, Frauen und Kinder, aber [Root.GetFirstName] zeigte kein Erbarmen. Nach [Root.GetHerHis]er schrecklichen Tat setzte [Root.GetSheHe] die Stadt in Brand und legte sie in Schutt und Asche.;;;;;;;;;;;x
```

Comment:
I'd say it should be "Terrible news have reached" instead of "has reached". Maybe ask Duke, since he's native speaker, but I'm pretty sure it should be "have reached".

Reply:
At one point I thought so too, but apparently "news" is always singular (https://www.espressoenglish.net/is-news-singular-or-plural/).

I'll let it sit until a native speaker has checked it, might be for the best.

Reply:
"Terrible news has reached your court" is correct. "Terrible news" is a singular noun referring the the general singular concept of bad news